extensions/amp-story/0.1/amp-story-hint.css

```diff
@@ -73,6 +73,10 @@
   display: none !important;
 }
 
+.show-first-page-overlay .i-amphtml-story-navigation-help-overlay {
```

Comment:
Is it an issue if you simply unset padding from the general `.i-amphtml-story-navigation-help-overlay` selector, as opposed to overriding?

Reply:
its basically that one case need it one doesn't so as more cases come up, may be then it's a good time to refactor this